src/bp/ui-studio/src/web/components/ChatEmulator/Emulator.jsx

```diff
@@ -76,23 +80,49 @@ export default class EmulatorChat extends React.Component {
     })
   }
 
+  getAxiosConfig() {
+    let axiosConfig = { params: { include: 'nlu,state,suggestions,decision,credentials' } }
+
+    if (this.state.externalToken) {
+      axiosConfig = {
+        ...axiosConfig,
+        headers: {
+          ExternalAuth: `Bearer ${this.state.externalToken}`
```

Comment:
in the backend I read `externalauth` and here you send it as `ExternalAuth` will this work ? do axios is _smart_ enough to make a copy to make a small case copy of each header ?

Reply:
HTTP headers are normally sent by the browser in upper camel case, but axios headers are in lower case. I wanted to stay consistent with existing convention